tests/test_repocard.py

```diff
@@ -127,7 +127,7 @@
       value: 0.2662102282047272
       name: Accuracy
       config: default
-      verified: false
+      verified: true
```

Comment:
For simplicity I decided to make this dummy eval "verified" so I could reuse the existing test cases of `ModelCardTest`. Let me know if you'd prefer a separate test class :)